### PR TITLE
Use quayio-pull-through-cache

### PR DESCRIPTION
**Pull request by jmguzik**
Signed-off-by: Jakub Guzik <jguzik@redhat.com>
/hold

not sure if this is the correct place

### CODE REVIEW

**Comment by jmguzik** [comment]
Analyzed failures, none of them related to images pull through failure.
@jupierce we can enable it once you think it's ready.
/hold
/cc @jupierce

File: ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh [hunk 1]
```diff
@@ -182,14 +182,46 @@ EOF
 
 yq-go m -x -i "${CONFIG}" "${PATCH}"
 
+oc registry login
+# ocp_version=4.9 4.10 etc.
+ocp_version=$(oc adm release info ${RELEASE_IMAGE_LATEST} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+
+# excluding older releases because of the bug fixed in 4.10, see: https://bugzilla.redhat.com/show_bug.cgi?id=1960378
+if (( ocp_minor_version > 9 )); then
```
Comment (jupierce) [inline ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh:191]: or major > 4 to future proof

**Comment by jupierce** [comment]
One nit, but it can wait if desired.
/lgtm

**Pull request merged**